src/Core/Grid/Definition/Factory/CartRuleGridDefinitionFactory.php

```diff
@@ -0,0 +1,262 @@
+                        ),
+                    ])
+                    ),
+            ])
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters(): FilterCollection
+    {
+        return (new FilterCollection())
+            ->add((new Filter('id_cart_rule', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('ID', [], 'Admin.Global'),
+                ],
+            ])
+            ->setAssociatedColumn('id_cart_rule')
+            )
+            ->add((new Filter('name', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('Name', [], 'Admin.Global'),
+                ],
+            ])
+            ->setAssociatedColumn('name')
+            )
+            ->add((new Filter('priority', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('Priority', [], 'Admin.Global'),
+                ],
+            ])
+            ->setAssociatedColumn('priority')
+            )
+            ->add((new Filter('code', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('Code', [], 'Admin.Global'),
+                ],
+            ])
+            ->setAssociatedColumn('code')
+            )
+            ->add((new Filter('quantity', TextType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('Quantity', [], 'Admin.Catalog.Feature'),
+                ],
+            ])
+            ->setAssociatedColumn('quantity')
+            )
+            ->add((new Filter('date_to', DateRangeType::class))
+            ->setTypeOptions([
+                'required' => false,
+                'attr' => [
+                    'placeholder' => $this->trans('Expiration date', [], 'Admin.Catalog.Feature'),
+                ],
+                'date_format' => 'YYYY-MM-DD HH:mm:ss',
+            ])
+            ->setAssociatedColumn('date_to')
+            )
+            ->add((new Filter('active', YesAndNoChoiceType::class))
+            ->setAssociatedColumn('active')
+            )
+            ->add((new Filter('actions', SearchAndResetType::class))
+            ->setAssociatedColumn('actions')
+            ->setTypeOptions([
+                'reset_route' => 'admin_common_reset_search_by_filter_id',
+                'reset_route_params' => [
+                    'filterId' => self::GRID_ID,
+                ],
+                'redirect_route' => 'admin_cart_rules_index',
+            ])
+            ->setAssociatedColumn('actions')
+            )
+            ;
```

Comment:
strange ident. Does cs fixer do that ?

Reply:
nevermind, I checked other grids and it looks the same (even tho i remember it was different), so its not worth your time. It must be cs fixer issue.